src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,70 @@
+.header{
```

Comment:
```suggestion
.header {
```

Have you added sass?

Reply:
yes vaia. It was added while bootstrapping the project.